modules/migration-installing-migration-operator-olm.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id="installing-migration-operator-with-olm_{context}"]
+= Installing the CAM Operator with OLM
+
+You can install the CAM Operator on the {product-title} 4 cluster with OLM. The CAM Operator installs the Cluster Application Migration tool on this cluster by default.
+
+.Procedure
+
+. Click *Administration* -> *Namespaces*.
+. On the *Namespaces* page:
+.. Click *Create Namespace*.
+.. Enter `openshift-migration` in the *Name* field and click *Create*.
+
+. Click *Operators* -> *OperatorHub*.
+. On the *OperatorHub* page:
+.. Scroll or type a keyword into the *Filter by keyword* field (in this case, *Migration*) to find the *Cluster Application Migration Operator*.
+.. Select the *Cluster Application Migration Operator* and click *Install*.
+
+. On the *Create Operator Subscription* page:
+.. Select the *openshift-migration* namespace if it is not already selected.
+.. Select an *Automatic* or *Manual* approval strategy.
+.. Click *Subscribe*.
+
+. Click *Catalog* -> *Installed Operators*. The *Cluster Application Migration Operator* is listed in the *openshift-migration* project with the status *InstallSucceeded*.
+
+. On the *Installed Operators* page:
+
+.. Under *Provided APIs*, click *View 12 more...*.
+.. Click *Create New* -> *MigrationController*.
+
+.. If you are installing the CAM Operator on an {product-title} _4.1_ cluster, add the following parameter:
++
+----
+spec:
+  deprecated_cors_configuration: true
+----
+
+.. If you are installing the CAM Operator on an {product-title} 4 _source_ cluster, update the following parameter values so that the migration controller and CAM web console are not installed:
++
+----
+spec:
+  ...
+  migration_controller: false
+  migration_ui: false
+----
+
+.. Click *Create*.
+
+. If you installed the migration controller on this cluster:
+
+.. Click *Workloads* -> *Pods* to verify that the controller manager, migration UI, Restic, and Velero pods are running.
+.. Click *Networking* -> *Routes*. In the `openshift-migration` namespace, the CAM host URL is the URL listed under *Location* if the CAM is running on this cluster. You will use the CAM host URL to configure cross-origin resource sharing on the other clusters and to launch the CAM web console.
+
+. If you did not install the migration controller, click *Workloads* -> *Pods* to verify that the Rustic and Velero pods are running.
```

Comment:
Throughout the PR, "pods" -> "Pods"

Reply:
"Pods" is capitalized? Why?! It's a generic term. 

Did a quick search in openshift/modules. 
"pods" - 923 times in 272 files.
"Pods" - 354 times in 153 files.

But I'll change it, since maybe this is a recent style decision. Seems odd to me.

Reply:
Is this only for named pods (e.g., Restic Pods) or for all pods (e.g., static Pods)?

Reply:
All instances of Pod/Pods should be uppercase according to our guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#pod

I agree, I don't like this guidance either, but it was decided at least a year ago (before I joined the OpenShift docs team) so I follow it.